Rajitha/recursion-practice.js

```diff
@@ -0,0 +1,53 @@
+/****************************************************
+* Create a function to get the factoral of a number.
+* Input: factoral(5)  Output: 120
+****************************************************/
+var factorial = function(number) {
+  if (number <= 0) {
+    return 1;
+  } else {
+    return (number * factorial(number - 1));
+  }
+};
+factorial(6);
+
+/*********************************************************
+* Create a function to get the power of a number.
+* Multiply the first input x(second input) number of times
+* Input: power(2, 5)
+* Output: 32
+* reason: 2x2x2x2x2
+**********************************************************/
+var numPower = function(num,exp) {
+  if (exp === 0){
+    return 1;
+  } else {
+    return num * numPower(num,exp-1);
+  }
+};
+numPower(2,5);
```

Comment:
same as above.... thinking of other possible number combinations. 
-  numPower(100, 1).
-  numPower(....